packages/cli-plugin-metro/src/commands/runPackager/index.ts

```diff
@@ -0,0 +1,18 @@
+import {runPackager} from './runPackager';
+
+export default {
+  name: 'run-packager-hook',
+  description: 'Starts the packager',
+  func: runPackager,
+  options: [
+    {
+      name: '--port <number>',
+      description: 'Port to run packager on.',
+    },
+    {
+      name: '--terminal <string>',
+      description:
+        'Launches packager in a new window using the specified terminal path.',
```

Comment:
**Main feedback ⬇️**

Interesting, can we merge this all into the `start` command — aligning with your proposal in https://github.com/react-native-community/discussions-and-proposals/pull/613? Specifically:

- The improved error message behaviour when `start` (RN CLI `start`) is called and something is already on the configured Metro port.
- `react-native start --terminal` switches modes to integrate the `startServerInNewWindow.js` behaviour.

Then, we only expose `startCommand` and it can be used by `doctor`, `run-android`, `run-ios`.

cc @thymikee Does this seem feasible?